src/set_theory/cardinal/basic.lean

```diff
@@ -207,10 +208,14 @@ theorem out_embedding {c c' : cardinal} : c ≤ c' ↔ nonempty (c.out ↪ c'.ou
 by { transitivity _, rw [←quotient.out_eq c, ←quotient.out_eq c'], refl }
 
 theorem lift_mk_le {α : Type u} {β : Type v} :
-  lift.{(max v w)} (#α) ≤ lift.{max u w} (#β) ↔ nonempty (α ↪ β) :=
+  lift.{max v w} (#α) ≤ lift.{max u w} (#β) ↔ nonempty (α ↪ β) :=
 ⟨λ ⟨f⟩, ⟨embedding.congr equiv.ulift equiv.ulift f⟩,
  λ ⟨f⟩, ⟨embedding.congr equiv.ulift.symm equiv.ulift.symm f⟩⟩
```

Comment:
If you add
```lean
theorem lift_mk_le_mk {α : Type u} {β : Type (max u v)} : lift.{v} (#α) ≤ #β ↔ nonempty (α ↪ β) :=
by rw [← lift_mk_le', lift_id', lift_umax]
```
here, then (see below)

Reply:
I'm not sure about this. If this were added, it would only make sense to also have the version for `#α ≤ lift.{u} (#β)`, and then these same theorems for `=` and `<`. I don't think there's a need, given that all these variants are easily provable from the original, plus one or two extra `lift` lemmas. Even if I wanted to add them, I think they'd constitute scope creep.

Also, #15041 should simplify the proof in question somewhat, by allowing us to get rid of some universe annotations.